### PR TITLE
Link to Knowledge Sharing Sessions

### PR DESCRIPTION
**Pull request by jafeltra**
**Description:** This PR adds a section to the README that links to the Knowledge Sharing Sessions that would be helpful to watch before starting development. I reworked a bit of the README to fit the section in better as well.

**Testing Instructions:** Make sure the new section makes sense and is where you'd expect it.

**Related Issue:** N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,15 +92,22 @@ To run the SUSHI Build task, use VS Code's Run Task feature. The "Run Task" feat
 
 Note that you must have SUSHI installed locally in order for the task to run successfully. See [SUSHI Installation instructions](https://fshschool.org/docs/sushi/installation/) for help installing.
 
-## Compile and Run (for Developers)
+## Instructions for Developers
+
+### Intro to FSH VS Code Extension Development
+
+To learn more about the FSH VS Code Extension, watch the Knowledge Sharing Sessions for [Developing FSH Tools](https://vimeo.com/990594107/c35c3f5e43) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/256514908/KSS%203%20-%20Developing%20FSH%20Tools.pdf)) and [Developing FSH VS Code Extension](https://vimeo.com/1008502402/e0d365dc49) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/265095834/KSS%207%20-%20Developing%20FSH%20VS%20Code%20Extension.pdf)).
```
Comment (cmoesel) [inline README.md:99]: As noted in similar PRs, the Developing FSH Tools video link needs to be fixed:
```suggestion
To learn more about the FSH VS Code Extension, watch the Knowledge Sharing Sessions for [Developing FSH Tools](https://vimeo.com/990594228/056b5c075f) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/256514908/KSS%203%20-%20Developing%20FSH%20Tools.pdf)) and [Developing FSH VS Code Extension](https://vimeo.com/1008502402/e0d365dc49) (view the slides [here](https://confluence.hl7.org/display/FHIR/FSH+Knowledge+Sharing+Sessions?preview=/256509612/265095834/KSS%207%20-%20Developing%20FSH%20VS%20Code%20Extension.pdf)).
```

**Review by cmoesel — CHANGES_REQUESTED** [review]
Everything looks good, with the exception of the wrong Developing FSH Tools link. Once that's fixed, I think we're good to go!

File: README.md (hunk 1 above)
Comment (jafeltra) [reply README.md]: Fixed in [eb82d62](https://github.com/FHIR/vscode-fsh/pull/98/commits/eb82d62c219c6ab5d2b373716c20c5170b818ad1).

**Review by cmoesel — APPROVED** [review]
Looks good!

**Pull request merged**